raml/schemas/mongo/analysis.json

```diff
@@ -4,6 +4,13 @@
     "type": "object",
     "properties": {
     	"_id":          {"type": "string"},
+        "parent":       {
+                            "type": "object",
+                            "properties": {
+                                "type": "string",
+                                "id":   "string"
+                            }
+                        },
```

Comment:
It would also need `permissions`, `public` and `archived`, correct? Are we even using the mongo schemas for analysis? These have kind of fallen out of maintenance as new development didn't use them.